userspace/libsinsp/sinsp_filtercheck_event.cpp

```diff
@@ -724,7 +724,7 @@ uint8_t* sinsp_filter_check_event::extract_abspath(sinsp_evt* evt, uint32_t* len
 			dirfdarg = "newdir";
 			patharg = "newpath";
 		}
-	} else if(etype == PPME_SYSCALL_UNLINKAT_E || etype == PPME_SYSCALL_UNLINKAT_2_X) {
+	} else if(etype == PPME_SYSCALL_UNLINKAT_2_X) {
```

Comment:
With this PR, we introduced the possibility for dirfd and path to be empty. For this reason, in the below code, after having performed the `evt->get_param(dirfdargidx)` and `evt->get_param(pathargidx)`, we should check that these parameters are not empty. Something like this:
```c++
const auto dirfd_param = evt->get_param(dirfdargidx);
const auto path_param = evt->get_param(pathargidx);
if (dirfd_param->empty() || path_param->empty()) {
    return 0;
}

const auto dirfd = dirfd_param->as<int64_t>();
const auto path = path_param->as<std::string_view>();
```